app/src/test/java/com/techcourse/dao/UserDaoTest.java

```diff
@@ -38,7 +36,7 @@ void findAll() {
 
     @Test
     void findById() {
-        final var user = userDao.findByAccount("gugu").get();
+        final var user = userDao.findById(1L).get();
 
         assertThat(user.getAccount()).isEqualTo("gugu");
     }
```

Comment:
```suggestion
@Test
void findById() {
    assertThat(userDao.findById(1L))
        .isPresent() // Optional이 비어있지 않은지 확인
        .hasValueSatisfying(user -> // 그 안에 있는 값(user)이
            assertThat(user.getAccount()).isEqualTo("gugu") // 특정 조건을 만족하는지 확인
        );
}
```

사소함: 이런 식으로 AssertJ가 제공하는 Optional 전용 assertion을 활용해보는 것은 어떨까요?

Reply:
옹, 이런게 있었군요!